docs/In-Band Manageability Developer Guide.md

```diff
@@ -729,7 +648,6 @@ triggering query commands via manifest returns query results through dynamic tel
 |:------------------------|:-------------------------------------|
 | [all]                   | Publish all the  details together    |
 | [fw]                    | Publish firmware details             |
-| [guid]                  | Publish GUID of Vision card          |
 | [hw]                    | Publish hardware details             |
 | [os]                    | Publish operating system details     |
 | [security]              | Publish security details             |
```

Comment:
should be able to also remove security.  I think that was only for BC.

Reply:
@nmgaston removed